src/test/checkTest.js

```diff
@@ -0,0 +1,10 @@
+import chai, { expect } from 'chai'
+
+
+  describe('check', () => {
+
+    it('does the thing', () => {
+    })
+
+
+  })
```

Comment:
What is this file? The filename does not lead to anything else committed and has no logic yet. Omit from PRs.